fuel-async/fuel-rx3/src/main/kotlin/fuel/RxJavaExtension.kt

```diff
@@ -0,0 +1,15 @@
+package fuel
+
+import io.reactivex.rxjava3.core.Single
+import okhttp3.Call
+import okhttp3.Response
+import java.io.IOException
+
+public fun Call.toSingle(): Single<Response> =
+    Single.defer {
```

Comment:
[q] I wonder whether we should favor `create` over defer so we can call `cancel` properly with `call.cancel()`